### PR TITLE
[en] Update outdated update-readme-doc

### PR DESCRIPTION
**Pull request by madhumita-kundo**
**Changes :**

Update Swagger specification to `Swagger specification(currently known as OpenAPI specification) ` in README.md file for more clarity.

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
Hi @madhumita-kundo, and thank you for your fix! Could you please sign the CLA before your PR can be reviewed.

For steps to sign the CLA please refer: https://github.com/kubernetes/community/blob/master/CLA.md

**Review by Arhell — COMMENTED** [review]
/easycla

**Comment by madhumita-kundo** [comment]
> Hi @madhumita-kundo, and thank you for your fix! Could you please sign the CLA before your PR can be reviewed.
> 
> For steps to sign the CLA please refer: https://github.com/kubernetes/community/blob/master/CLA.md

It is done now...thank you @dipesh-rawat

**Comment by madhumita-kundo** [comment]
> /easycla

It is done now...thank you @Arhell

File: README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ This will start the local Hugo server on port 1313. Open up your browser to <htt
 
 ## Building the API reference pages
 
-The API reference pages located in `content/en/docs/reference/kubernetes-api` are built from the Swagger specification, using <https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs>.
+The API reference pages located in `content/en/docs/reference/kubernetes-api` are built from the Swagger specification (currently known as OpenAPI specification), using <https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs>.
```
Comment (NitishKumar06) [inline README.md:73]:
```suggestion
The API reference pages located in `content/en/docs/reference/kubernetes-api` are built from the Swagger specification, also known as OpenAPI specification that is using <https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs>.
```

**Review by NitishKumar06 — COMMENTED** [review]
PTAL!

**Comment by madhumita-kundo** [comment]
> Sorry for the failure it's because of my changes. If you want it to work:
> 
> 1. [Synchronize your fork with upstream](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork)
> 2. Get the new changes on main: `git fetch && git checkout main && git pull origin main`
> 3. Rebase your branch on main `git checkout update-readme-doc && git rebase -i main`

Thank you very much @mtardy ....it's working now...

File: README.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ This will start the local Hugo server on port 1313. Open up your browser to <htt
 
 ## Building the API reference pages
 
-The API reference pages located in `content/en/docs/reference/kubernetes-api` are built from the Swagger specification, using <https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs>.
+The API reference pages located in `content/en/docs/reference/kubernetes-api` are built from the Swagger specification, also known as OpenAPI specification that is using <https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs>.
```
Comment (divya-mohan0209) [inline README.md:73]:
```suggestion
The API reference pages located in `content/en/docs/reference/kubernetes-api` are built from the Swagger specification, also known as OpenAPI specification, using <https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs>.
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Added commas + amended wording.

**Comment by madhumita-kundo** [comment]
> Added commas + amended wording.

Hi @divya-mohan0209 ...I have updated the pr with suggested changes...thank you

**Review by dipesh-rawat — COMMENTED** [review]
All the suggested fixes have been applied. Looks good to me.

**Comment by madhumita-kundo** [comment]
> All the suggested fixes have been applied. Looks good to me.

Thank you @dipesh-rawat ...what should be my next course of action?

File: README.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ This will start the local Hugo server on port 1313. Open up your browser to <htt
 
 ## Building the API reference pages
 
-The API reference pages located in `content/en/docs/reference/kubernetes-api` are built from the Swagger specification, using <https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs>.
+The API reference pages located in `content/en/docs/reference/kubernetes-api` are built from the Swagger specification, also known as OpenAPI specification, using <https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs>.
```
Comment (bradtopol) [inline README.md:73]: Great job!

**Comment by bradtopol** [comment]
Great job!
/lgtm
/approve

**Pull request merged**